test/test_deployment.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+# Service entity tests
+class TestService < MiniTest::Test
+  def test_delete_a_deployment_with_propagation_policy_query
+    value_propagation_policy = 'Foreground'
+    namespace = 'my_namespace'
+    deployment = 'my_deployment'
+
+    stub_request(:get, %r{/api/extensions/v1beta1$})
+      .to_return(body: open_test_file('v1beta1_api_resource_list.json'), status: 200)
+    stub_request(:delete, %r{/namespaces/#{namespace}/deployments/#{deployment}})
+      .to_return(body: open_test_file('entity_list.json'), status: 200)
```

Comment:
Is this what DELETE actually returns?  https://kubernetes.io/docs/api-reference/v1.5/#delete-24 (clicking "curl" on top right) says it returns:
```
{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Success",
  "code": 200
}
```